include/services/error_indexes.h

```diff
@@ -389,6 +389,7 @@ enum ErrorID
     ErrorSQLstmtHandle = -90044,                                        /*!< ErrorSQLstmtHandle */
     ErrorOnFileOpen = -90045,                                           /*!< Error on file open */
     ErrorOnFileRead = -90046,                                           /*!< Error on file read */
+    ErrorNullByteInjection = -90047,
```

Comment:
You also should add description for the error in file services/kernel/error_handling.cpp